Jenkinsfile.deployment

```diff
@@ -0,0 +1,33 @@
+pipeline {
+    environment {
+        ARTIFACTORY_URL = credentials('ARTIFACTORY_URL')
+    }
+
+    agent {
+        label 'build'
+    }
+
+    tools {
+        gradle "gradle-7.2"
+        jdk 'openjdk17'
+    }
+    
+
+    stages {
+   
+    
+        stage ('Build Libraries') {
+            steps {
+                withCredentials([usernamePassword(credentialsId: 'artifactoryuserpass', usernameVariable: 'ARTIFACTORY_USER', passwordVariable: 'ARTIFACTORY_PASSWORD')]) { 
```

Comment:
Probably need to build this more, test coverage, sonar ext

Reply:
Done